devicetypes/smartthings/zwave-water-temp-light-sensor.src/zwave-water-temp-light-sensor.groovy

```diff
@@ -0,0 +1,226 @@
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ *  Z-Wave Water/Temp/Light Sensor
+ *
+ *  Author: SmartThings
+ *  Date: 2018-08-09
+ */
+
+metadata {
+	definition(name: "Z-Wave Water/Temp/Light Sensor", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "x.com.st.d.sensor.moisture", runLocally: true, minHubCoreVersion: '000.017.0012', executeCommandsLocally: false) {
+		capability "Water Sensor"
+		capability "Sensor"
+		capability "Battery"
+		capability "Health Check"
+		capability "Temperature Measurement"
+		capability "Illuminance Measurement"
+
+		fingerprint mfr: "019A", prod: "0003", model: "000A", deviceJoinName: "Sensative Strips"
+	}
+
+	simulator {
+		status "dry": "command: 3003, payload: 00"
+		status "wet": "command: 3003, payload: FF"
+		status "dry notification": "command: 7105, payload: 00 00 00 FF 05 FE 00 00"
+		status "wet notification": "command: 7105, payload: 00 FF 00 FF 05 02 00 00"
+		status "wake up": "command: 8407, payload: "
+	}
+
+	tiles(scale: 2) {
+		multiAttributeTile(name: "water", type: "generic", width: 6, height: 4) {
+			tileAttribute("device.water", key: "PRIMARY_CONTROL") {
+				attributeState("dry", icon: "st.alarm.water.dry", backgroundColor: "#ffffff")
+				attributeState("wet", icon: "st.alarm.water.wet", backgroundColor: "#00A0DC")
+			}
+		}
+		valueTile("temperature", "device.temperature", inactiveLabel: false, width: 2, height: 2) {
+			state "temperature", label:'${currentValue}°'
+		}
+		valueTile("illuminance", "device.illuminance", inactiveLabel: false, width: 2, height: 2) {
+			state "luminosity", label:'${currentValue} ${unit}', unit:"lux"
+		}
+		valueTile("battery", "device.battery", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "battery", label: '${currentValue}% battery', unit: ""
+		}
+
+		main "water"
+		details(["water", "temperature", "illuminance", "battery"])
+	}
+}
+
+def installed() {
+	// Dome Leak Sensor sends WakeUpNotification every 12 hours. Please add zwaveinfo.mfr check when adding other sensors with different interval.
+	sendEvent(name: "checkInterval", value: (2 * 12 + 2) * 60 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+	def cmds = [ zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x01).format(),
+				zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x03).format(),
+				zwave.notificationV3.notificationGet(notificationType: 0x05).format(), //water alarm
+                zwave.batteryV1.batteryGet().format()]
+	response(cmds)
+}
+
+def updated() {
+	// Dome Leak Sensor sends WakeUpNotification every 12 hours. Please add zwaveinfo.mfr check when adding other sensors with different interval.
+	sendEvent(name: "checkInterval", value: (2 * 12 + 2) * 60 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
```

Comment:
We started extracting send checkInterval event to one method. Feel free to do the same if you like. If something is wrong we need to change only one place.